tests/conformance_tests/tests/conformance_runner.rs

```diff
@@ -45,12 +45,14 @@ lazy_static! {
         Regex::new(r"test-vectors/corpus/vm_violations/x--state_mutation--after-transaction").unwrap(),
         Regex::new(r"test-vectors/corpus/vm_violations/x--state_mutation--readonly").unwrap(),
 
-        // These tests are out of date. Ideally they would be updated in the test-vectors repo. 2022-02-03
-        // Updated vectors have been temporarily put in `extra-vectors/`
         Regex::new(r"test-vectors/corpus/specs_actors_v6/TestMinerWithdraw/withdraw_from_non-owner_address_fails").unwrap(),
         Regex::new(r"test-vectors/corpus/specs_actors_v6/TestAggregateBadSender").unwrap(),
         // This test fails even after being updated.
         Regex::new(r"extra-vectors/TestAggregateBadSender/8466b548087bb6c8c8469b4135521b147364ed7625467c8ac149f8785abcab5d").unwrap(),
+
+        // We're too strict with these vectors:
```

Comment:
What does it mean that we're too strict? Is it an issue with `fvm-test-vectors` or Forest? Should it be raised to someone from FVM?

Reply:
Not sure. Someone will have to look into this eventually.

Reply:
Do you think we could add a task with some relevant logs and link it here?

Reply:
I've added an issue: https://github.com/ChainSafe/forest/issues/1457